packages/gatsby/src/images/search/ico-npm.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="34" height="13" viewBox="0 0 34 13" xmlns="http://www.w3.org/2000/svg">
+<svg width="34" height="17" viewBox="0 0 34 9" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I noticed that the text wasn't centered so it caused alignment issue with `display: flex; align-items: center;`